### PR TITLE
adding docs for qa-ing flags

### PR DESCRIPTION
**Pull request by hhargreaveseppo**

### CODE REVIEW

File: docs/feature-flagging/qa-flags.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# QA Flags
+
+## Testing Environments
+
+Eppo supports separation of feature flags and their allocations between environments. Create an Eppo environment specifically for Staging or QA following this [documentation](/sdks/api-keys). Environments are separated by [SDK keys](/sdks/api-keys), and no unique key will be shared across Eppo Environments. Once an Environment is created, create an SDK for that Environment.
```
Comment (emetelka) [inline docs/feature-flagging/qa-flags.md:5]: /feature-flags/index#environments is a better place to link to. It has all this information

File: docs/feature-flagging/qa-flags.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# QA Flags
```
Comment (emetelka) [inline docs/feature-flagging/qa-flags.md:1]: I think there's a better title we can use. Maybe "Testing Feature Flags"

File: docs/feature-flagging/qa-flags.md
Comment (emetelka) [inline docs/feature-flagging/qa-flags.md:1]: I think this should go in the guides section

File: docs/feature-flagging/qa-flags.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+![Screenshot of list of Environments](/img/feature-flagging/qa-flag-environments.png)
+
+Click **Add Allocation** on the flag and serve the treatment or the experience to be tested to 100%.
+
+Set your flag to **On** to have it run in the selected environment.
+
+![Screenshot of allocation for Staging Environment](/img/feature-flagging/qa-flag-staging-allocation.png)
+
+## Testing in Production
```
Comment (emetelka) [inline docs/feature-flagging/qa-flags.md:19]: You may also want to mention that the user can copy their allocation from testing to production

**Review by emetelka — CHANGES_REQUESTED** [review]

File: docs/feature-flagging/qa-flags.md
Comment (hhargreaveseppo) [reply docs/feature-flagging/qa-flags.md:1]: gotcha, moved it over

File: docs/feature-flagging/qa-flags.md (hunk 1 above)
Comment (hhargreaveseppo) [reply docs/feature-flagging/qa-flags.md]: fixed

File: docs/feature-flagging/qa-flags.md (hunk 3 above)
Comment (hhargreaveseppo) [reply docs/feature-flagging/qa-flags.md]: included that, great suggestion

File: docs/feature-flagging/qa-flags.md (hunk 2 above)
Comment (hhargreaveseppo) [reply docs/feature-flagging/qa-flags.md]: fixed

**Pull request merged**